docuilib/src/components/UILivePreview.module.scss

```diff
@@ -1,13 +1,24 @@
 $preview-width: 350px;
```

Comment:
Change the device width based on what Alexey wants

Reply:
Done